docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -365,271 +613,273 @@ is a dictionary which contains the following keys:
+is set as the ``:email`` property of an ``inet:whois:contact`` node and tag the associated FQDN as sinkholed
+(``#cno.infra.dns.sink.holed``). Because the trigger is a simple one, you want it to run inline (i.e., you do
+not need the ``--async`` option).
 
-- **Housekeeping.** You created a trigger to automatically look up and apply geolocation and autonomous system
-  (AS) properties to IPv4 nodes when they are created in Synapse. However, you already have a large number of
-  IPv4 nodes that existed before the trigger was added. You can create a one-time cron job to retrieve and
-  "backfill" this information for IPv4s that already exist.
+.. storm-cli:: trigger.add prop:set --name 'Tag sinkholed FQDNs based on known sinkholer email' --prop inet:whois:contact:email --query { +{ :email-> inet:email +#cno.infra.dns.sink.hole } -> inet:whois:rec -> inet:fqdn [ +#cno.infra.dns.sink.holed ] }
 
-- **Process intensive jobs.** Data enrichment may be resource intensive where it generates a significant number
-  of write operations. If you reguarly perform routine (non-urgent) enrichment, it can be scheduled to run
-  when it will have less impact on users.
+.. TIP::
   
+  For ``prop:set`` and ``prop:del`` triggers, the node whose property caused the trigger to fire is sent
+  inbound to the trigger's Storm.
 
-Usage Notes
-~~~~~~~~~~~
-
-- Users must be granted permissions in order to be able to work with cron jobs (i.e., to execute the
-  associated ``cron.*`` Storm commands).
-
-- Cron jobs execute **with the permissions of the user who created the job**. If a cron job calls a
-  macro, the macro will execute with the permissions of the cron job (macros execute with the
-  permissions of the calling user).
+We can view the newly created trigger using ``trigger.list``:
 
-- Cron jobs **reside** in the Cortex but **execute** within a particular :ref:`gloss-view`. This has
-  implications for environments that use multiple views or that regularly :ref:`gloss-fork` and later
-  merge or delete views. For example:
-  
-  - Cron jobs that execute in a **forked view** are "orphaned" when you merge or delete the fork.
-    The jobs will remain in Synapse (because they reside in the Cortex), but will not execute because
-    they are not assigned to a view. You must assign the jobs to a new view for them to run (or delete
-    them if no longer needed).
+.. storm-cli:: trigger.list
 
-- When viewing cron jobs (i.e., with the :ref:`storm-cron-list` command), Synapse returns
-  **all cron jobs in the Cortex,** regardless of the view the job executes in.
+The output of ``trigger.list`` includes the following columns:
 
-- Cron jobs are exclusive - if for some reason a job has not finished executing before its next scheduled
-  start, the original job will run to completion and the "new" job will be skipped.
+- **user** - The user that the trigger runs as (typically the user who created the trigger).
```

Comment:
The corresponding list of descriptions for the `cron.list` output does not capitalize the first letter in the description

Reply:
(not sure which one should be fixed)

Reply:
lowercase for consistency